routers/api/v1/repo/pull.go

```diff
@@ -366,6 +367,10 @@ func EditPullRequest(ctx *context.APIContext, form api.EditPullRequestOption) {
 	}
 	if form.State != nil {
 		if err = issue.ChangeStatus(ctx.User, ctx.Repo.Repository, api.StateClosed == api.StateType(*form.State)); err != nil {
+			if models.IsErrDependenciesLeft(err) {
+				ctx.Error(http.StatusPreconditionFailed, "", fmt.Sprintf("cannot close this pull request because it still has open dependencies"))
```

Comment:
Why fmt.Sprintf when there are no `%` verbs?

Reply:
Good question. Probably just because I copied it from somwhere else... :grin:

Should I use `fmt.Sprint` or just put the string directly?

Reply:
Just put the string directly.